File: 2st/Yeonwoo/길이에 따른 연산.py
def solution(num_list):
    answer = 0
    num_list_len = len(num_list)
    if num_list_len >= 11:
        for i in num_list:
            answer+=i
    elif num_list_len <= 10:
        answer = 1
        for i in num_list:
            answer *= i
    return answer

File: 4st/Seohyun/programmer/길이에 따른 연산.js
function solution(num_list) {
  let answer = num_list.length >= 11 ? 0 : 1;

  if (num_list.length >= 11) {
    for (const i of num_list) {
      answer += i;
    }
  } else {
    for (const i of num_list) {
      answer *= i;
    }
  }
  return answer;
}
